.github/tools/cioutput.py

```
@@ -0,0 +1,669 @@
+
+def main() -> int:
+	"""Main function for CLI usage.
+
+	Returns:
+		Exit code (0 for success, non-zero for failure)
+	"""
+	try:
+		tool = configure_output_tool()
+		if tool:
+			return 0
+	except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)